packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -48,11 +48,6 @@
   /// @group link
   $link-visited: $ibm-color__purple-60 !default !global;
 
-  /// @type Color
-  /// @access public
-  /// @group link
-  $link-inverse-color: #6ea6ff !default !global;
```

Comment:
Removing this because it isn't used anywhere else, outside of `Tooltip` (and is now replaced)

Reply:
@jendowns should we add a deprecation notice in case folks are using it? Since it has `@access public` (not too sure since I'm unfamiliar with this component, but seems like it could cause an issue potentially)

Reply:
OH good point! So in that case, I'll restore the token & add a deprecation notice so folks know to use `$inverse-link` instead 👍

Reply:
@joshblack I made the change you suggested here 👍